### PR TITLE
Upgrade to Polkadot 0.9.10

### PR DESCRIPTION
**Pull request by echevrier**

### CODE REVIEW

File: polkadot-launch/launch-rococo-local-with-integritee.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
 	"relaychain": {
-		"bin": "../../bin/polkadot-0.9.8",
+		"bin": "../../paritytech/polkadot/target/release/polkadot",
```
Comment (brenzi) [inline polkadot-launch/launch-rococo-local-with-integritee.json:3]: this appears inconsistent with the other paths

**Review by brenzi — CHANGES_REQUESTED** [review]
please do NOT update hard-coded spec files! They need to stay exactly the same
So please un-add all changes to res/....json

File: polkadot-parachains/shell-runtime/src/lib.rs [hunk 2]
```diff
@@ -35,28 +35,26 @@ use sp_std::prelude::*;
...
 pub use sp_runtime::BuildStorage;
 pub use sp_runtime::{Perbill, Permill};
-
-pub use pallet_balances::Call as BalancesCall;
-
 // XCM imports
-use frame_support::traits::Filter;
-use xcm::v0::{Junction::*, MultiLocation, MultiLocation::*, NetworkId};
+use frame_support::traits::Contains;
+//use xcm::v0::{Junction::*, MultiLocation, MultiLocation::*, NetworkId};
```
Comment (brenzi) [inline polkadot-parachains/shell-runtime/src/lib.rs:56]: please remove commented, outdated lines

File: polkadot-parachains/shell-runtime/src/lib.rs [hunk 3]
```diff
@@ -298,7 +296,10 @@ pub type XcmOriginToTransactDispatchOrigin = (
 );
 
 match_type! {
-	pub type JustTheParent: impl Contains<MultiLocation> = { X1(Parent) };
+	pub type JustTheParent: impl Contains<MultiLocation> = {
+		//X1(Parent)
```
Comment (brenzi) [inline polkadot-parachains/shell-runtime/src/lib.rs:300]: dito

File: polkadot-launch/launch-rococo-local-with-integritee.json (hunk 1 above)
Comment (echevrier) [reply polkadot-launch/launch-rococo-local-with-integritee.json:3]: my bad: my local settings. I will remove it

File: polkadot-parachains/integritee-runtime/src/lib.rs [hunk 4]
```diff
@@ -58,9 +58,11 @@ pub use sp_runtime::{Perbill, Permill};
 pub use pallet_teerex::Call as TeerexCall;
 
 // XCM imports
+use frame_support::traits::Everything;
 use pallet_xcm::{EnsureXcm, IsMajorityOfBody, XcmPassthrough};
 use polkadot_parachain::primitives::Sibling;
-use xcm::v0::{BodyId, Junction::*, MultiAsset, MultiLocation, MultiLocation::*, NetworkId, Xcm};
+//use xcm::v0::{BodyId, Junction::*, MultiLocation, MultiLocation::*, NetworkId};
```
Comment (clangenb) [inline polkadot-parachains/integritee-runtime/src/lib.rs:64]: same as brenzies comment below. please remove the comment. :)

File: polkadot-launch/launch-rococo-local-with-integritee.json (hunk 1 above)
Comment (brenzi) [reply polkadot-launch/launch-rococo-local-with-integritee.json:3]: I'll do

**Review by brenzi — APPROVED** [review]

**Pull request merged**